### DISCUSSION TITLE
Serenity IIS Publish and Deployment

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5314] [Help]
Serenity Projemi tamamladım visual studio da yayımla işlemini yapıyorum ve daha sonra IIS yükledikten sonra hata alıyorum.
![image](https://user-images.githubusercontent.com/24252615/102753249-f11a4080-437b-11eb-9403-86f97bcb4f73.png)
projemi IIS' e Dağıtım yapana kadar hangi adımları yapmam gerekiyor yardımcı olabilirmisiniz

**Comment by VictorTomaili** [comment]
Your Sql instance is not accessible. check your published connection string

**Reply by havvanur160401046** [reply]
There is no connectionstring in the web config file.
Could it be due to this.

**Reply by VictorTomaili** [reply]
It's seems to me, it's a .Net Core app. So web config must be in your appsettings.json file. 

Also check documentation from microsoft about deploying .net core app to IIS 
https://docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-iis?view=aspnetcore-3.1&tabs=visual-studio

**Reply by havvanur160401046** [reply]
![image](https://user-images.githubusercontent.com/24252615/102772622-e3bf7f00-4398-11eb-9ac5-f771f09dff6a.png)
I edited appsetting json but now I get this error

**Reply by VictorTomaili** [reply]
Please search error message on google first. It's not serenity related exception. Your IIS app pool user doesn't have access rights to your database.

**Reply by havvanur160401046** [reply]
I solved the problem thank you very much

**Reply by JohnRanger** [reply]
HI @havvanur160401046 ,

you have opened 2 discussions about the same problem. This is not best practice. I have given you a longer information in your other issue. Please check that one.

With kind regards,

john
